packages/terra-toolkit-docs/src/terra-dev-site/tool/eslint-config-terra/About.a.tool.mdx

```diff
@@ -0,0 +1,47 @@
+import { Badge } from '@cerner/eslint-config-terra/package.json?dev-site-package';
+
+<Badge />
+
+# Eslint Config Terra
+
+This configuration reflects Terra's supported eslint policy for javascript.
+
+Features:
+
+- extends the  [eslint-config-airbnb](https://github.com/airbnb/javascript/tree/main/packages/eslint-config-airbnb), which exports ESLint rules, including rules for ECMAScript 6+ and React.
+- checks for proper ReactJS Hooks usage via the [eslint-plugin-react-hooks](https://reactjs.org/docs/hooks-rules.html).
+- checks syntax for targeted browsers support via the [eslint-plugin-compat](https://github.com/amilajack/eslint-plugin-compat). It is recommended to use Terra's targeted browsers which are specified by the [browserslist-config-terra](https://github.com/cerner/terra-toolkit/tree/main/packages/browserslist-config-terra) module.
+- defines the browser and jest environment
+- defines enzyme globals for `shallow`, `render` and `mount` for jest testing
+
+## What is Eslint
+
+[ESlint](https://eslint.org/) is a pluggable linting utility for JavaScript and JSX.
+
+## Installation
+
+Install the module
+
+```shell
+npm install eslint --save-dev
+npm install @cerner/eslint-config-terra --save-dev
+```
+
+## Usage
+
+First, include the configuration defined by `@cerner/eslint-config-terra` via the  [extends](https://eslint.org/docs/user-guide/configuring#extending-configuration-files) property.
+
+Then, [define the browsers](https://github.com/amilajack/eslint-plugin-compat#targeting-browsers) for the `eslint-plugin-compat` plugin to target by add the browsers list configuration in your package.json through the `browserslist` key.
+
+### package.json
+
+```json
+{
+  "browserslist": [
+    "extends @cerner/browserslist-config-terra"
+  ],
```

Comment:
We should drop this in favor of the `browserslistrc` to avoid teams accidentally running into the browserslist dependency issues we ran into:

https://engineering.cerner.com/terra-ui/dev_tools/cerner/browserslist-config-terra/upgrade-guide

Reply:
good call